utils/check_repo.py

```diff
@@ -1054,7 +1053,7 @@ def ignore_undocumented(name: str) -> bool:
 
 def check_all_objects_are_documented():
     """Check all models are properly documented."""
-    documented_objs = find_all_documented_objects()
+    documented_objs, documented_methods_map = find_all_documented_objects()
```

Comment:
Note: I intend to use this new mapping in `check_docstrings`, to perform basic quality checks on the docstring of publicly documented methods